content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,145 +124,114 @@ enum Level {
 }
 ```
 
-This data model will be used to store incoming requests from your Worker.
+This data model will be used to fetch logs from your Worker.
 
-## 5. Create a repository and push to GitHub
+## 3. Create a repository and push to GitHub
 
 To prepare for the steps ahead, let's [create a private repository](https://github.com/new) on GitHub.
-
-![Create a private repository](../../../doc-images/guides/deploy/create-repo.png)
-
 Next, initialize your repository, then push your changes up to GitHub.
 
 ```terminal
-git remote add origin https://github.com/<username>/prisma-mongodb-cloudflare
+git remote add origin https://github.com/<username>/prisma-postgresql-cloudflare
 git add .
 git commit -m "initial commit"
 git push -u origin main
 ```
 
 You're ready to import your project into the Prisma Data Platform.
 
-## 6. Import your Project into the Prisma Data Platform
-
-With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with the [Data Proxy](/data-platform/classic-projects/data-proxy).
-
-1. To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
-
-   <Admonition type="info">
-
-   **Note**<br /><br />
-   A GitHub account is required to sign up for the Prisma Data Platform.
-
-   </Admonition>
-
-   ![Signup for Prisma Data Platform](../../../doc-images/data-platform/pdp-signup.png)
+## 4. Enable Accelerate in Prisma Data Platform
 
-2. Click **New Project** and then select **Import a Prisma repository**.
-3. Fill in the repository and project details, and click **Next**.
+With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with [Prisma Accelerate](/data-platform/accelerate).
```

Comment:
I believe CF workers now do have TCP support... wondering if we can rephrase this so that it becomes clearer?

Reply:
Prisma does not work on CF Workers yet. Can link to https://github.com/prisma/prisma/issues/21394